### DISCUSSION TITLE
FR: Edit form entries

### DISCUSSION THREAD

**Discussion by freshface** [statamic/cms#10807] [General]
Possibilty to allow form entries be editable thru the control panel

**Comment by duncanmcclean** [comment]
We already have a feature request open for this on our `ideas` repository - feel free to give it a 👍: https://github.com/statamic/ideas/issues/128

**Reply by freshface** [reply]
👌🏻 but its from 5 years ago :)

**Reply by duncanmcclean** [reply]
We sort feature requests by 👍 reactions. The FR doesn't have any reactions yet, so it's not a high priority.
